File: RememberMe/app/src/main/java/com/example/rememberme/SecondActivity.java
package com.example.rememberme;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class SecondActivity extends AppCompatActivity {

    private TextView mtvText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);
        mtvText=findViewById(R.id.textView);

        String name=PreferenceHelper.getFromPreference(SecondActivity.this,"username");
        mtvText.setText(name);
    }
}

File: NewsApplication/app/src/main/java/com/example/newsapplication/ui/SecondActivity.kt
package com.example.newsapplication.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.activity.viewModels
import com.bumptech.glide.Glide
import com.example.newsapplication.R
import com.example.newsapplication.data.models.Article
import com.example.newsapplication.databinding.ActivitySecondBinding
import com.example.newsapplication.viewModel.NewsViewModel

class SecondActivity : AppCompatActivity() {
    private lateinit var binding:ActivitySecondBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivitySecondBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val article = intent.getSerializableExtra("article") as Article

//        binding.apply {
//            newsHeadline.text = article?.title
//            newsDate.text = article?.publishedAt
//            newsSite.text = article?.author
//            tvContentSecond.text = article?.content
//        }
        binding.article = article
        Glide.with(binding.newsImage).load(article?.urlToImage).into(binding.newsImage)
    }
}